ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/event-search/EventSearch.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import { useState } from 'react';
+import { Button } from '@inngest/components/Button';
+import { useClient } from 'urql';
+
+import Input from '@/components/Forms/Input';
+import { useEnvironment } from '@/queries';
+import { EventTable } from './EventTable';
+import { searchEvents } from './searchEvents';
+import type { Event } from './types';
+
+const day = 1000 * 60 * 60 * 24;
+
+type Props = {
+  environmentSlug: string;
+};
+
+export function EventSearch({ environmentSlug }: Props) {
+  const [events, setEvents] = useState<Event[]>([]);
+  const [fetching, setFetching] = useState(false);
+  const [{ data: environment }] = useEnvironment({ environmentSlug });
+  const client = useClient();
+
+  async function onSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    setFetching(true);
+
+    try {
+      const form = new FormData(event.currentTarget);
+      const text = form.get('text');
+      if (typeof text !== 'string') {
+        // Should be unreachable
+        throw new Error('name must be a string');
+      }
+
+      if (!environment) {
+        // Should be unreachable
+        throw new Error('missing environment');
+      }
+
+      setEvents(
+        await searchEvents({
+          client,
+          environmentID: environment.id,
+          lowerTime: new Date(Date.now() - 3 * day),
+          text,
+          upperTime: new Date(),
```

Comment:
Is this UTC?

Reply:
JS date objects contain the timezone. JSON serialization turns them into ISO UTC:

<img width="406" alt="image" src="https://github.com/inngest/inngest/assets/20100586/2277fc8f-5e59-4018-9d86-9b4671bb1f3f">